src/app/(BottomTabNavigation)/AllCases/index.tsx

```diff
@@ -1,16 +1,16 @@
 import { router } from 'expo-router';
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useContext } from 'react';
 import { FlatList, Text, View, TouchableOpacity } from 'react-native';
 
 import styles from './styles';
 import Camera from '../../../../assets/camera.svg';
 import CaseCard from '../../../Components/CaseCard/CaseCard';
 import { CaseContext } from '../../../context/CaseContext';
-import supabase from '../../../supabase/createClient';
 
 function CasesScreen() {
-  const { allCases, loading } = useContext(CaseContext);
-
+  //const { allCases } = useContext(CaseContext);
+  const { activeCases, loading } = useContext(CaseContext);
+  const { inactiveCases } = useContext(CaseContext);
```

Comment:
Right now, inactive cases don't immediately show up on the `AllCases` screen unless the app is otherwise reloaded. I'm not sure the best way to fix it, but let's try adding a navigation listener that requires the cases when the page is visited. 
<img width="357" alt="Screen Shot 2024-03-20 at 5 51 18 PM" src="https://github.com/calblueprint/impact-fund/assets/66931067/fce7cf50-4643-4fbb-9811-8c3459bca16a">